projects/systelab-translate/src/test/number-format.pipe.spec.ts

```diff
@@ -72,10 +72,10 @@ describe('Pipe: NumberFormatPipe', () => {
 			.toBe('< 3.33%');
 	});
 
-	it('check number format - invalid rounding', () => {
-		expect(pipe.transform(3.3323335, '1dsfas3333423422', '%', '<'))
-			.toBe('');
-	});
+	// it('check number format - invalid rounding', () => {
```

Comment:
Why is need to comment this?

Reply:
Restored, it was a check to see if the issue with Travis was related to the unique test which was showing an error message in the local console.